Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1819,7 +1829,7 @@ namespace Remotion.BocAutoCompleteReferenceValue
         const availableWidth = position.left + referenceWidth - marginLeft;
         // js rounding errors sometimes create linebreaks
         // therefore we add a single pixel for the field to always be wide enough:
-        const requiredWidth = contentWidth + 1;
+        const requiredWidth = contentWidth + 1 + (requiredHeight >= maxHeightSafe ? scrollbarWidth : 0);
```

Comment:
do you want to add an explanatory comment here or just rely on the commit-linked issue and common sense for the explanation of this logic?

Reply:
Hmm I could also make it more readable by introducing a boolean `requireScrollbar`. That should make it require as little common sense as possible.